addon/services/modal.js

```diff
@@ -31,7 +31,7 @@ export default class Modal extends Service {
   @action
   add(item) {
     this.items.pushObject(item);
-    item.set('renderIndex', this.renderIndex);
+    item.renderIndex = this.renderIndex;
     this.renderIndex++;
```

Comment:
Can you combine these?
```suggestion
    item.renderIndex = this.renderIndex++;
```